### PR TITLE
Created feedback encouragement message

### PR DESCRIPTION
**Pull request by amitag2**
For issue #2845 

This is what the message looks like:
image

### CODE REVIEW

File: website/src/components/Chat/EncourageMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Badge } from "@chakra-ui/react";
+
+// kept brief so that it doesnt distract/interfere with user experience
+export const EncourageMessage = () => {
+  return (
+    <Badge ml="1" fontFamily="Geneva" fontWeight="normal" fontSize="0.4em" background="gray.300" color="black">
+      Thoughts? Let us know!
```
Comment (AbdBarho) [inline website/src/components/Chat/EncourageMessage.tsx:7]: can you move this to the localization?

**Review by AbdBarho — COMMENTED** [review]
Can you also provide a screenshot how it looks on mobile? thanks

Also in dark mode.

**Comment by notmd** [comment]
I'm gonna take over this

**Review by notmd — APPROVED** [review]
Thanks

**Pull request merged**